common/lib/xmodule/xmodule/lti_module.py

```diff
@@ -841,6 +855,17 @@ def get_client_key_secret(self):
                 return key, secret
         return '', ''
 
+    def is_past_due(self):
+        """
+        Is it now past this problem's due date, including grace period?
+        """
+        due_date = self.due  # pylint: disable=no-member
+        if self.graceperiod is not None and due_date:  # pylint: disable=no-member
```

Comment:
Can you help me here  – where do the `self.due` and `self.graceperiod` attributes come from? I see them listed in a few other modules, like in the `CapaModule`, but not in the `LTIFields`.